package.json

```diff
@@ -26,8 +26,14 @@
     "@aws-cdk/aws-lambda": "^1.115.0",
     "@aws-cdk/aws-rds": "^1.115.0",
     "@aws-cdk/aws-secretsmanager": "^1.115.0",
-    "@aws-cdk/core": "1.115.0",
+    "@aws-cdk/core": "^1.115.0",
+    "@aws-cdk/pipelines": "^1.115.0",
+    "apollo-server": "^3.1.2",
```

Comment:
I think we can remove all the packages below     `"@aws-cdk/pipelines": "^1.115.0",` I think those are only needed in api/

Reply:
oops keep `source-map-support` though.

Reply:
im guessing you had to add those when testing this to get the API to build with all npm packages? I think we can change this to be two pipelines one for API changes and one for cdk changes.

Reply:
I couldnt get it to build until I added those

Reply:
it was failing in codebuild FWIW